distributed/tests/test_client.py

```diff
@@ -4876,6 +4876,7 @@ async def test_recreate_task_futures(c, s, a, b):
     assert function(*args, **kwargs) == 2
 
 
+@pytest.mark.xfail(reason="This is actually using a Series for a key somewhere")
```

Comment:
Should we look into this before merging and breaking things?

Reply:
done. not a series as key, just a wrong `pytest.raises`